web-ui/src/components/pulse/Pulse.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
```

Comment:
This is a new component that displays a title, a row of sentiment icon buttons, and a text input for entering a comment.